ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureCloudConfig.java

```diff
@@ -71,6 +83,119 @@
       "com.github.ambry.cloud.azure.ConnectionStringBasedStorageClient";
   public static final String USE_ASYNC_AZURE_APIS = "use.async.azure.apis";
 
+  public static final String AZURE_STORAGE_ACCOUNT_INFO_STR = "storageAccountInfo";
+  public static final String AZURE_STORAGE_ACCOUNT_INFO_NAME = "name";
+  public static final String AZURE_STORAGE_ACCOUNT_INFO_PARTITION_RANGE = "partitionRange";
+  public static final String AZURE_STORAGE_ACCOUNT_INFO_STORAGE_SCOPE = "storageScope";
+  public static final String AZURE_STORAGE_ACCOUNT_INFO_STORAGE_ENDPOINT = "storageEndpoint";
+  public static final String AZURE_STORAGE_ACCOUNT_INFO_STORAGE_CONNECTION_STRING = "storageConnectionString";
+
+  /**
+   * Stores Azure Storage Account related info for one storage account.
+   */
+  public static class StorageAccountInfo {
+    private final String name;
+    private final int partitionRangeStart;
+    private final int partitionRangeEnd;
+    private final String storageScope;
```

Comment:
Should clientid, secret and authority also be part of this class?

Reply:
No, they are not specific to storage accounts